### ISSUE TITLE
Add support for touch portal

### ISSUE THREAD

**Issue by cj2tech** [KazWolfe/XIVDeck#47]
It would be nice to have support for [https://www.touch-portal.com](https://www.touch-portal.com) support since you already have the majority of the work done in ffxiv

**Comment by KazWolfe** [comment]
Hi and thank you for your request! At present, I don't have the resources to dedicate to porting my plugin to other systems. I will keep this open to track support of the Touch Portal though, in case I can ever come back to it in the future.

It does look like the [Touch Portal API](https://www.touch-portal.com/api/index.php) does carry most of what I care about, so it should be doable once I get around to buying the app and setting up my dev environment for it. 

In the meantime, Elgato's Stream Deck Mobile app can also be used for mobile-based control although I understand purchasing *yet another* application may be undesirable.

**Comment by cj2tech** [comment]
Thanks for the recommendation. I have tried the mobile version of Stream Deck and was not really happy with how restricted it is. But here is hoping to you getting around to porting it someday. Love your app and would love to see it on there.

**Comment by Teflondom25** [comment]
> Hi and thank you for your request! At present, I don't have the resources to dedicate to porting my plugin to other systems. I will keep this open to track support of the Touch Portal though, in case I can ever come back to it in the future.
> 
> It does look like the [Touch Portal API](https://www.touch-portal.com/api/index.php) does carry most of what I care about, so it should be doable once I get around to buying the app and setting up my dev environment for it.
> 
> In the meantime, Elgato's Stream Deck Mobile app can also be used for mobile-based control although I understand purchasing _yet another_ application may be undesirable.


@KazWolfe 

I've messed around with porting some SD projects over to touch portal, care if i fork this and give it a go?

**Comment by KazWolfe** [comment]
@Teflondom25 Not at all! Feel free to drop a link too and I'll include it in the README.

I'd _prefer_ if you just implemented the frontend side and still link to my backend server/plugin, but if you want to make a full fork, I won't stop you either. That said, I'm also happy to work with people to implement extra whatevers that would be useful for a specific platform.
